README.md

```diff
@@ -190,6 +190,14 @@ you can look through the [issue tracker][it].
 [it]: https://github.com/amethyst/amethyst/issues
 [gfi]: https://github.com/amethyst/amethyst/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
 
+## Sponsors
+Amethyst is supported by:
+<p>
+  <a href="https://www.digitalocean.com?utm_medium=opensource&utm_source=amethyst/">
+    <img src="https://opensource.nyc3.cdn.digitaloceanspaces.com/attribution/assets/SVG/DO_Logo_horizontal_blue.svg" width="201px">
+  </a>
+</p>
```

Comment:
I don't think the `<p>` make much of a difference since markdown is supposed to render a paragraph when it encounters one or more blank lines.

```
<p>Hello</p>

Sweet

<p>World</p>
```

#### Rendered
<p>Hello</p>

Sweet

<p>World</p>

Reply:
yup had the same thought but it seemed to work so I guess its fine 🤷‍♂️